qt/python/mantidqt/widgets/algorithmselector/test/test_algorithmselector.py

```diff
@@ -69,6 +69,9 @@ def test_include_hidden_algorithms(self):
         self.assertEqual(mock_get_algorithm_descriptors.mock_calls[-1], call(True))
 
 
+createDialog_func = 'mantidqt.interfacemanager.InterfaceManager.createDialogFromName'
```

Comment:
For clarity is may be worth using the full function name for this variable name and making it clear this is a function name not a function.

e.g. `createDialogFromName_function_name`